src/fsharp/SyntaxTree.fsi

```diff
@@ -761,8 +761,11 @@ type SynExpr =
     /// F# syntax: if expr then expr
     /// F# syntax: if expr then expr else expr
     | IfThenElse of
+        ifKeyword: SynIfThenElseStart *
```

Comment:
Perhaps

    ifKeyword: range *
    isElif: bool *

is enough, without SynIfThenElseStart?